sdk/storage/azure-storage-blob/azure/storage/blob/_shared/authentication.py

```diff
@@ -64,6 +67,9 @@ def _get_verb(self, request):
 
     def _get_canonicalized_resource(self, request):
         uri_path = urlparse(request.http_request.url).path
+        if isinstance(request.context.transport, AioHttpTransport):
```

Comment:
These changes must be replicated in queues and files too. since it's a part of the _shared folder.

Reply:
Okay, will modify file and queues with these changes.